.github/maintainers_guide.md

```diff
@@ -24,23 +24,33 @@ Run this from the root of the project
 
 ### Testing
 
-Test can be run directly with Deno.
+Test can be run directly with Deno:
 
-```
-deno test src
-```
+    deno task test
+
+You can also run a test coverage report with:
 
+    deno task coverage
 
 ### Releasing
 
-* Releases for this library are triggered off of tags. You should create a new GitHub Release, and use a new version that adheres to semantic versioning based on what's being release. Version tags should match the following pattern: `1.0.1` (no `v` preceding the number).
-* Once the release has been created, a corresponding version will be avaliable on https://deno.land/x/deno_slack_api.
+Releases for this library are automatically generated off of git tags. To create a new release:
+
+1. Create a new GitHub Release from the [Releases page](https://github.com/slackapi/deno-slack-api/releases) by clicking the "Draft a new release" button.
+2. Input a new version manually into the "Choose a tag" input. Ensure that this version adheres to [semantic versioning][semver] based on what's being released. Version tags should match the following pattern: `1.0.1` (no `v` preceding the number).
+  - After you input the new version, click the "Create a new tag: x.x.x on publish" button.
+3. Set the "Target" input to the "main" branch.
```

Comment:
nit: Should we call out anything about ensuring that everything on `main` is in a releasable state?

Reply:
I added a note about this in the intro sentence to this section.